public/ge0.html

```diff
@@ -76,29 +86,33 @@
 </head>
 
 <body>
-  <div id="header">
-    <span id="name">${name}</span>
-    <span>
-      <a id="open" href="om:/${path}" class="link">
-        Open in Organic Maps
-      </a>
-      <a href="https://omaps.app/get" target="_blank" class="link">
-        Install Organic Maps
-      </a>
-      <a href="https://www.openstreetmap.org/?mlat=${lat}&mlon=${lon}#map=${zoom}/${lat}/${lon}" target="_blank" class="link">
-        See on OpenStreetMap
-      </a>
-    </span>
+  <div id="header" class="box">
+    <div id="open" class="block">
+      <a href="om:/${path}" class="link">Show in Organic Maps</a>
+    </div>
+    <div id="osm" class="block">
+      <a href="https://www.openstreetmap.org/?mlat=${lat}&mlon=${lon}#map=${zoom}/${lat}/${lon}" target="_blank" class="link">See on OpenStreetMap</a>
+    </div>
+    <div class="block">
+      <a href="https://omaps.app/get" target="_blank" class="link">Get Organic Maps</a>
+    </div>
   </div>
   <div id="map" class="map"></div>
+
   <script type="text/javascript">
     const isiOS = navigator.platform.substr(0, 2) === 'iP' ||  // iPhone, iPad, iPod, including simulators.
       (navigator.userAgent.includes('Mac') && 'ontouchend' in document);  // iPad on iOS 13.
     const isAndroid = !isiOS && /(android)/i.test(navigator.userAgent);
     const isDesktop = !isiOS && !isAndroid;
+    coords = 'Lat: ${lat}, Lon: ${lon}';
```

Comment:
```suggestion
    let coords = '${lat}, ${lon}';
```

Reply:
Are you sure, we should show coordinates without any keywords? I tried it first, but looks a bit ugly....